docs/patterns/tools/use-tools/use-specific-actions.mdx

```diff
@@ -10,112 +10,124 @@ description: "Each Tool (like GitHub, Slack, etc.) comes with many Actions. You
 </Tip>
 
 ### Specifying Actions
+
 `GITHUB_CREATE_AN_ISSUE` and `GITHUB_COMMIT_EVENT` are action IDs for actions in the GitHub Tool
+
 <CodeGroup>
-```python Python
-from composio import ComposioToolSet, App, action
+  ```python Python
+  from composio_langchain import ComposioToolSet, Action
 
-toolset = ComposioToolSet()
+  toolset = ComposioToolSet()
 
-# can pass multiple actions
-tools = toolset.get_tools(actions=[Action.GITHUB_CREATE_AN_ISSUE,Action.GITHUB_COMMIT_EVENT])
-```
+  # can pass multiple actions
+  tools = toolset.get_tools(
+      actions=[Action.GITHUB_CREATE_AN_ISSUE]
+  )
+  ```
 
-```javascript Javascript
-import { LangchainToolSet } from "composio-core";
+  ```javascript Javascript
+  import { LangchainToolSet } from "composio-core";
 
-const toolset = new LangchainToolSet();
+  const toolset = new LangchainToolSet();
 
-// can pass multiple actions
-const tools = await toolset.getTools({actions: ['github_issues_create','github_commit_event']});
-```
+  // can pass multiple actions
+  const tools = await toolset.getTools({actions: ['github_issues_create','github_commit_event']});
+  ```
 </CodeGroup>
 
 ### Filtering Actions
+
 Actions can be filtered by tags or use case.
+
 <Tabs>
-<Tab title="Filtering Actions by Tags">
-Filter the Actions in an App based on tags.
-For example, you can:
-- Filter all user-related Actions using the "users" tag
-- Retrieve metadata-related Actions using the "meta" tag
+  <Tab title="Filtering Actions by Tags">
+    Filter the Actions in an App based on tags.
+    For example, you can:
 
-<CodeGroup>
-```python Python
-from composio_langchain import ComposioToolSet, App
-tool_set = ComposioToolSet()
+    * Filter all user-related Actions using the "users" tag
```

Comment:
The docstring for `find_actions_by_tags` method (lines 1044-1050) incorrectly references use case parameters instead of tag parameters. Consider updating the docstring to accurately reflect the tag-based filtering functionality.